src/command/render/render-contexts.ts

```diff
@@ -507,31 +507,31 @@ async function resolveFormats(
       (isHtmlOutput(format, true) || isHtmlDashboardOutput(format)) &&
       formatHasBootstrap(projFormat) && projectTypeIsWebsite(projType)
     ) {
-      if (formatHasBootstrap(inputFormat)) {
-        if (
-          inputFormat.metadata[kTheme] !== undefined &&
-          !ld.isEqual(inputFormat.metadata[kTheme], projFormat.metadata[kTheme])
-        ) {
-          warnOnce(
-            `The file ${file.path} contains a theme property which is being ignored. Website projects do not support per document themes since all pages within a website share the website's theme.`,
-          );
-        }
-        delete inputFormat.metadata[kTheme];
-      }
-      if (formatHasBootstrap(directoryFormat)) {
-        if (
-          directoryFormat.metadata[kTheme] !== undefined &&
-          !ld.isEqual(
-            directoryFormat.metadata[kTheme],
-            projFormat.metadata[kTheme],
-          )
-        ) {
-          warnOnce(
-            `The file ${file.path} contains a theme provided by a metadata file. This theme metadata is being ignored. Website projects do not support per directory themes since all pages within a website share the website's theme.`,
-          );
-        }
-        delete directoryFormat.metadata[kTheme];
-      }
+      // if (formatHasBootstrap(inputFormat)) {
+      //   if (
+      //     inputFormat.metadata[kTheme] !== undefined &&
+      //     !ld.isEqual(inputFormat.metadata[kTheme], projFormat.metadata[kTheme])
+      //   ) {
+      //     warnOnce(
+      //       `The file ${file.path} contains a theme property which is being ignored. Website projects do not support per document themes since all pages within a website share the website's theme.`,
+      //     );
+      //   }
+      //   delete inputFormat.metadata[kTheme];
+      // }
+      // if (formatHasBootstrap(directoryFormat)) {
+      //   if (
+      //     directoryFormat.metadata[kTheme] !== undefined &&
+      //     !ld.isEqual(
+      //       directoryFormat.metadata[kTheme],
+      //       projFormat.metadata[kTheme],
+      //     )
+      //   ) {
+      //     warnOnce(
+      //       `The file ${file.path} contains a theme provided by a metadata file. This theme metadata is being ignored. Website projects do not support per directory themes since all pages within a website share the website's theme.`,
+      //     );
+      //   }
+      //   delete directoryFormat.metadata[kTheme];
+      // }
```

Comment:
In relation with my other comments, do I understand correctly quarto will now no more consider that project-level bootstrap theme wins over any document theme ? 

This is mainly to understand correctly, and be sure we document correctly this change, as potentially "breaking" change, and give advices for users on how to adjust in their website.

Reply:
> do I understand correctly quarto will now no more consider that project-level bootstrap theme wins over any document theme ?

You have it slightly wrong. It's not that project-level bootstrap won over any document theme. It's that document-level themes _were disabled_. They no longer need to be.

And then, yes: themes in documents now override themes in projects. This is now consistent with our general idea of configuration behavior defaults. From more general to more specific: ambient to project to document to code cells.

Reply:
You are right - I think I missed the disabled part. I was not really possible to do differently before so we disabled, and now that we can do it, we want to enable the document overrides. 

It can appear as a change in design and could lead to breaking change for anyone having configuration that were disabled previously. 

So I think we need to hightlight this so that users can check there website projects, and know what to do if they see undesired differences. 

Thank you for the explanation.